cognite_toolkit/cdf.py

```diff
@@ -256,85 +255,42 @@ def deploy(
     # Set environment variables from local.yaml
     read_environ_config(root_dir=build_dir, build_env=build_env, set_env_only=True)
 
-    if include and (invalid_types := set(include).difference(_AVAILABLE_DATA_TYPES)):
-        print(
-            f"  [bold red]ERROR:[/] Invalid data types specified: {invalid_types}, available types: {_AVAILABLE_DATA_TYPES}"
-        )
+    typer.echo(Panel(f"[bold]Deploying config files from {build_dir} to environment {build_env}...[/]"))
+    build_path = Path(build_dir)
+    if not build_path.is_dir():
+        typer.echo(f"  [bold red]WARNING:[/] {build_dir} does not exists. Did you forget to run `cdf-tk build` first?")
         exit(1)
 
-    include = include or list(_AVAILABLE_DATA_TYPES)
-    if interactive:
-        include = _select_data_types(include)
-
-    print(Panel(f"[bold]Deploying config files from {build_dir} to environment {build_env}...[/]"))
-    # Configure a client and load credentials from environment
-    if not Path(build_dir).is_dir():
-        alternatives = {
-            folder.name: f"{folder.parent.name}/{folder.name}"
-            for folder in Path(build_dir).parent.iterdir()
-            if folder.is_dir()
-        }
-        matches = difflib.get_close_matches(Path(build_dir).name, list(alternatives.keys()), n=3, cutoff=0.3)
-        print(
-            f"  [bold red]WARNING:[/] {build_dir} does not exists. Did you mean one of these? {[alternatives[m] for m in matches]}"
-        )
-        exit(1)
```

Comment:
I removed this as I think it is no longer helpful. The build directory should be `build` 99% of the time.